Gopkg.toml

```diff
@@ -44,7 +44,7 @@ required = [
 
 [[constraint]]
   name = "cloud.google.com/go"
-  version = "0.17.0"
+  version = "0.23.0"
```

Comment:
Do we need to constrain on exactly `0.23.0` (as opposed to at least?)

Reply:
Barring some sort of major future behavior-breakage I'd imagine we are fine to go with at least v0.23.0, changed in 21a3e27